tests/k4unit.q

```diff
@@ -61,50 +62,82 @@ KUltf:{ / (load test file) - load tests in csv file <x> into KUT
 	/KUT,:update file:x,action:lower action,lang:`q^lower lang,ms:0^ms,bytes:0j,repeat:1|repeat from `action`ms`lang`code`repeat`comment xcol("SISSI*";enlist .KU.DELIM)0:x:hsym x;
 	neg before-count KUT}
 
-KUltd:{ / (load test dir) - load all *.csv files in directory <x> into KUT
+KUltd:{ / (load test dir) - load all *.csv files in directory <x> into KUT except process.csv
 	before:count KUT;
-	files:(` sv)each(x,'key x);KUltf each files where(lower files)like"*.csv";
+	files:(` sv)each(x,'key x);KUltf each files where 1=sum (lower files) like/: ("*process.csv";"*.csv");
```

Comment:
I think it might be more clear/readable to be a little more verbose here:

```suggestion
	files:(` sv)each(x,'key x);KUltf each files where (lower[files]like"*.csv")&not lower[files]like"*process.csv";
```

Reply:
Think about this further... it might be better to modify `KUltf` to skip any CSVs that aren't in k4unit format, rather than hardcoding a single name to skip? This would allow e.g. defining some mock data in a CSV to be loaded for tests etc., which atm I think would probably cause a problem when `KUltf` tries to load differently formatted files?

Reply:
Yeah I've thought a little about this, as under this testing framework you could have various CSVs in the same folder. I'll give this a go